### PR TITLE
GG-36475-Update control scripts and JVM defaults

### PR DESCRIPTION
**Pull request by dy2285**
This is in response to @sk0x50 's comments and suggestions on a previously closed branch due to errors during rebasing:

1. License Comment: A minor case correction. Given that it's part of the standard license text, it's best to keep it consistent across all files. Changes have been made in similar situations in these files too.
2. Exit Command: Removed the 'exit 1' command after including 'jvmdefaults.sh' to ensure the script continues executing.
3. -XX:+AggressiveOpts: Reinstated the '-XX:+AggressiveOpts' option for Java versions prior to 11, as it was deprecated in JDK 11.
4. add-opens and add-exports: Ensured that all necessary 'add-opens' and 'add-exports' options are included for Java versions 11 and above to maintain compatibility.
5. Conditional add-opens: Updated the script to conditionally add 'add-opens' options based on the Java version.
6. Experimental Options: Restored the commented experimental options in 'control.sh' to allow users to enable them if needed.

### CODE REVIEW

File: bin/control.bat [hunk 1]
```diff
@@ -84,110 +84,69 @@ goto checkIgniteHome2
...
-:: Remote debugging (JPDA).
-:: Uncomment and change if remote debugging is required.
-:: set CONTROL_JVM_OPTS=-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=8787 %CONTROL_JVM_OPTS%
-::
-
-::
 :: Final CONTROL_JVM_OPTS for Java 9+ compatibility
-::
-call "%SCRIPTS_HOME%\include\jvmdefaults.bat" %MAJOR_JAVA_VER% "%CONTROL_JVM_OPTS%" CONTROL_JVM_OPTS
+if %JAVA_VERSION% GEQ 9 (
```
Comment (sk0x50) [inline bin/control.bat:146]: So, your idea is removing `jvmdefaults` call and adding specific options directly in this script, right?
Could you please clarify the exact option(s) required for the control script and not listed in the `jvmdefaults`?
Perhaps, we can add one or two lines to the `jvmdefaults` instead of copying the entire list to this script.

File: bin/control.bat [hunk 2]
```diff
@@ -224,3 +183,20 @@ del %RESTART_SUCCESS_FILE%
...
 goto :eof
+
+:getJavaSpecificOpts
+if %1 == 8 (
+    set CONTROL_JVM_OPTS=-XX:+AggressiveOpts %2
+    exit /b 0
+)
+
+if %1 GEQ 9 if %1 LSS 11 (
+    set CONTROL_JVM_OPTS=-XX:+AggressiveOpts --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED --illegal-access=permit --add-modules=java.xml.bind %2
```
Comment (sk0x50) [inline bin/control.bat:194]: Could you please list the options one per line? It is more readable and easier to review.

File: bin/include/jvmdefaults.bat [hunk 3]
```diff
@@ -24,74 +24,15 @@ set value=""
...
-    -XX:+AggressiveOpts ^
-    --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
-    --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
-    --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
-    --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
-    --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    --add-modules=java.xml.bind ^
-    %current_value%
+    set value=-XX:+AggressiveOpts --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED --add-exports=java.base/sun.nio.ch=ALL-UNNAMED --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED --illegal-access=permit --add-modules=java.xml.bind %current_value%
```
Comment (sk0x50) [inline bin/include/jvmdefaults.bat:31]: It looks like, you don't use this script to define jvm options for the control utility. So, what is the reason for this change?

**Comment by dy2285** [comment]
Thank you for your previous review @sk0x50. Due to the reorganization of the tasks and the subsequent creation of a clean branch, I am closing this PR in favor of a new one that has been created to address the specific issue of suppressing illegal reflective access warnings. The new PR has been assigned to Igor Belyakov for review, as it was indicated that a CS team member should review this change.  Thank you for your understanding.
